internal/website/content/howto/runtimevar/_index.md

```diff
@@ -116,7 +116,7 @@ open a variable stored in `etcd` via a URL, you can use the
 
 [etcd]: https://etcd.io/
 
-#### Constructor {#etcd-ctor}
+#### Etcd Constructor {#etcd-ctor}
```

Comment:
The spell it with lower case letter even in the beginning of the sentence or in the header:
https://github.com/etcd-io/etcd#etcd-tcp-ports
https://etcd.io/